### PR TITLE
App 205 pairing method extension

### PR DESCRIPTION
**Pull request by aryeko**

### CODE REVIEW

File: android/src/main/java/com/tuya/smart/rnsdk/activator/TuyaActivatorModule.kt [hunk 1]
```diff
@@ -39,6 +38,13 @@ import com.facebook.react.bridge.WritableMap
 import com.facebook.react.modules.core.DeviceEventManagerModule.RCTDeviceEventEmitter
 import com.tuya.smart.sdk.api.*
 
+object ScanBeamConstants{
+  const val UUID = "uuid"
+  const val DEVICE_TYPE = "uuid"
+  const val MAC = "uuid"
+  const val ADDRESS = "uuid"
+}
+
```
Comment (yahacom) [inline android/src/main/java/com/tuya/smart/rnsdk/activator/TuyaActivatorModule.kt]: probably it should have different values.
also better to move it under `com.tuya.smart.rnsdk.utils.Constant` and import from there

File: android/src/main/java/com/tuya/smart/rnsdk/activator/TuyaActivatorModule.kt [hunk 2]
```diff
@@ -107,6 +113,40 @@ class TuyaActivatorModule(reactContext: ReactApplicationContext) : ReactContextB
...
+        ScanBeamConstants.MAC, ScanBeamConstants.DEVICE_TYPE), params)) {
+
+      val multiModeActivatorBean = MultiModeActivatorBean();
+      multiModeActivatorBean.ssid = params.getString(SSID);
+      multiModeActivatorBean.pwd = params.getString(PASSWORD);
+
+      multiModeActivatorBean.uuid = params.getString("uuid")
+      multiModeActivatorBean.deviceType = params.getInt("deviceType")
+      multiModeActivatorBean.mac = params.getString("mac")
+      multiModeActivatorBean.address = params.getString("address")
```
Comment (yahacom) [inline android/src/main/java/com/tuya/smart/rnsdk/activator/TuyaActivatorModule.kt]: better to use constants imported from `com.tuya.smart.rnsdk.utils.Constant`

**Review by yahacom — APPROVED** [review]

File: android/src/main/java/com/tuya/smart/rnsdk/utils/Constant.kt [hunk 3]
```diff
@@ -36,6 +36,11 @@ object Constant {
     const val TEMPUNITENUM = "tempUnitEnum"
     const val REGION = "region"
 
+    const val UUID = "uuid"
+    const val DEVICE_TYPE = "uuid"
+    const val MAC = "uuid"
+    const val ADDRESS = "uuid"
+
```
Comment (yahacom) [inline android/src/main/java/com/tuya/smart/rnsdk/utils/Constant.kt]: again - values
```suggestion
    const val UUID = "uuid"
    const val DEVICE_TYPE = "deviceType"
    const val MAC = "mac"
    const val ADDRESS = "address"

```

**Review by yahacom — CHANGES_REQUESTED** [review]

File: android/src/main/java/com/tuya/smart/rnsdk/utils/Constant.kt (hunk 3 above)
Comment (aryeko) [reply android/src/main/java/com/tuya/smart/rnsdk/utils/Constant.kt]: What a miss 😅

**Pull request merged**